site/content/user-guide/amqp-adapter.md

```diff
@@ -38,6 +35,34 @@ There is a subtle difference between the *device identifier* (*device-id*) and t
 ### SASL EXTERNAL Authentication
 TODO
 
+## AMQP Command-line Client
+For the purposes of demonstrating the usage of the AMQP adapter, the **Hono CLI Module** contains an AMQP command-line client for interacting with the AMQP adapter. This module does not activate the AMQP client by default and should be activated using the `--spring.profile.active` argument.
+
+The command-line client contains the following arguments (with default values):
+* `message.address`: The AMQP 1.0 message address (default: `telemetry/DEFAULT_TENANT/4711`)
```

Comment:
why don't all params include the *double dashes*, i.e. `--message.address`? The `--spring.profiles.active` does ...